docs/KVSet/KVSet.v1.0.0.schema.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://azconfig.io/schemas/KVSet/v1.0.0/KVSet.json",
+  "title": "An Azure App Configuration schema for importing/exporting using 'appconfig/kvset' format",
+  "type": "object",
+  "required": [
+    "items"
+  ],
+  "properties": {
+    "items": {
+      "$id": "#/properties/items",
+      "type": "array",
+      "title": "Items",
+      "description": "An array of key-values in the Azure App Configuration store.",
+      "items": {
+        "$ref" : "#/$defs/kv"
+      }
+    }
+  },
+  "$defs": {
+    "kv": {
+      "type": "object",
+      "title": "Key value pair",
+      "description": "An individual key-value pair and associated metadata in the 'appconfig/kvset' format.",
+      "properties": {
+        "key": {
+          "$id": "#/properties/key",
+          "type": "string",
+          "title": "Key",
+          "description": "The 'key' in the key-value pair.",
+          "examples": [
+            "background_color"
+          ],
+          "pattern": "^(.*)$",
+          "minLength": 1
+        },
+        "value": {
+          "$id": "#/properties/value",
+          "type": "string",
+          "title": "Value",
+          "description": "The 'value' in the key-value pair.",
+          "examples": [
+            "blue"
+          ],
+          "pattern": "^(.*)$"
+        },
+        "label": {
+          "$id": "#/properties/label",
+          "title": "Label",
+          "description": "The label for the key-value pair.",
+          "examples": [
+            "staging"
+          ],
+          "default": null,
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "content_type": {
+          "$id": "#/properties/content_type",
+          "title": "Content Type",
+          "description": "The content type for the key-value pair.",
+          "examples": [
+            "plaintext"
+          ],
+          "default": null,
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "tags": {
+          "$id": "#/properties/tags",
+          "title": "Tags",
+          "type": "object",
+          "description": "The tags assigned to the key value pair.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "key",
+        "value"
```

Comment:
According to the REST API doc, only the key is required.
https://docs.microsoft.com/en-us/azure/azure-app-configuration/rest-api-key-value#set-key

Reply:
Updated the `required` field to contain only `key`.